demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -212,6 +218,108 @@ def is_image_path_valid(self) -> bool:
             return False
         return True
 
+    def verify_readme_image_paths(self) -> bool:
+        """ Validate readme (not pack readme) images relative and absolute paths.
+
+        Returns:
+            bool: True If all links both relative and absolute are valid else False.
+        """
+        # If there are errors in one of the following validations return False
+        if any([self.check_readme_relative_image_paths(),
+                self.check_readme_absolute_image_paths()]):
+            return False
+        return True
+
+    def check_readme_relative_image_paths(self, is_pack_readme: bool = False) -> list:
+        """ Validate readme images relative paths.
+            (1) prints an error if relative paths in the pack README are found since they are not supported.
+            (2) Checks if relative paths are valid (in other readme files).
+
+        Arguments:
+            is_pack_readme (bool) - True if the the README file is a pack README, default: False
+
+        Returns:
+            list: List of the errors found
+        """
+        error_list = []
+        error_code: str = ''
+        error_message: str = ''
+        should_print_error = not is_pack_readme  # If error was found, print it only if its not pack readme.
```

Comment:
code is self-explanatory,
comment should say ***why*** we do it

Reply:
Done